ament_copyright/test/test_copyright.py

```diff
@@ -28,6 +28,16 @@ def test_apache2_standard():
     assert rc == 0, 'Found errors'
 
 
+def test_apache2_cpp_multiline():
+    rc = main(argv=[os.path.join(cases_path, 'apache2_license_multiline_comment/case.cpp')])
```

Comment:
I'd prefer explicitly testing your function, not just seeing that main doesn't error out. The same for the XML test below.

Reply:
Once https://github.com/ament/ament_lint/pull/363 is merged I can also add tests to the file `ament_copyright/test/test_parser.py` that test the specific `get_multiline_comment_block` method.

Reply:
@audrow @methylDragon I added tests for the changes in this branch. Can you please take a look?